config/reasoner/op2dl/main/mapping.n3

```diff
@@ -0,0 +1,161 @@
+PREFIX : <#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX adres: <https://data.vlaanderen.be/ns/adres#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX crypto: <http://www.w3.org/2000/10/swap/crypto#>
+PREFIX e: <http://eulersharp.sourceforge.net/2003/03swap/log-rules#>
+PREFIX ex: <http://example.org/test#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX func: <http://www.w3.org/2007/rif-builtin-function#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX log: <http://www.w3.org/2000/10/swap/log#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX persoon: <https://data.vlaanderen.be/ns/persoon#>
+PREFIX persoon_loket: <http://data.vlaanderen.be/ns/persoon#>
+PREFIX skos:<http://www.w3.org/2004/02/skos/core#>
+PREFIX string: <http://www.w3.org/2000/10/swap/string#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+# DL and OP both use persoon:gebruikteVoornaam but with different semantics (!)
+# Hence the mapping is in two steps.
+# The first step maps the DL persoon:gebruikteVoornaam to the OP counterpart
+# and drops the DL persoon:gebruikteVoornaam statements with e:becomes
+# A second reasoning will map DL foaf:name to OP persoon:gebruikteVoornaam
+# Doing both at once would lead all DL foaf:names to incorrectly become foaf:givennames
+# in OP
+
+# First name
```

Comment:
I guess we don't expect persons right?

Reply:
Idd, it's just a generic mapping (I inverted the OP -> DL mapping). There's no performance drawback and keeps the reasoner config generic and re-usable